app/konnect/runtime-manager/composite-runtime-groups/conflicts.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Conflicts in runtime groups
+content_type: reference
+badge: enterprise
+---
+
+A conflict is a potential issue in composite configuration. 
+For example, there might be a name conflict between services, or the existance of consumer credentials, 
+which become available to all members of a composite group.
+
+Once you have at least one runtime instance connected to your composite runtime group, 
+you may see one of the following error messages:
+
+Composite runtime group:
+
+```
+Conflicts have been detected between these runtime groups: 
+<group name>
+<group name>
+```
+
+![Composite runtime group invariances](/assets/images/docs/konnect/konnect-invariances-composite.png)
+
+![Standard runtime group invariances](/assets/images/docs/konnect/konnect-invariances-members.png)
```

Comment:
Should this one move to below the Standard runtime group error message?

Reply:
@cloudjumpercat, this was existing layout, and I'm reading as first one shows the error, second one shows what happens after you click View to see more details. make sense?

Reply:
@acgoldsmith After looking at the output, what you have right now makes sense.